src/main/java/no/nav/veilarbaktivitet/internapi/InternApiService.java

```diff
@@ -77,7 +77,7 @@ public AktivitetData hentAktivitet(Integer aktivitetId) {
     private List<AktivitetData> filtrerKontorsperret(List<AktivitetData> aktiviteter) {
         return aktiviteter
                 .stream()
-                .filter(a -> a.getKontorsperreEnhetId() == null)
+                .filter( a -> authService.sjekKvpTilgang(a.getKontorsperreEnhetId()))
```

Comment:
er denne nullsafe?

Reply:
jau, men den sjekker ikke for tom string slik som stringutils.isempty() gjør

Reply:
endrer sjekkKvpTilgang til å ta i bruk det jeg